lib/dolly/document.rb

```diff
@@ -51,7 +51,8 @@ def rev= value
       doc['_rev'] = value
     end
 
-    def save
+    def save options = {}
+      return false unless options[:validate] == false || valid?
```

Comment:
@rubenrails that was my response to @netlogics regarding the validation stuff.  I hadn't responded to you yet since you had not answered yet.  I agree with both you and @netlogics reasonings to included the options.  I was waiting to respond to you after hearing you reasons, as they may have actually been different than @netlogics response